common/xrdp_client_info.h

```diff
@@ -57,11 +59,11 @@ struct xrdp_client_info
   char hostname[32];
   int build;
   int keylayout;
-  char username[256];
-  char password[256];
-  char domain[256];
-  char program[256];
-  char directory[256];
+  char username[INFO_CLIENT_MAX_CB_LEN];
+  char password[INFO_CLIENT_MAX_CB_LEN];
+  char domain[INFO_CLIENT_MAX_CB_LEN];
+  char program[INFO_CLIENT_MAX_CB_LEN];
+  char directory[INFO_CLIENT_MAX_CB_LEN];
```

Comment:
This fix is a good catch. However, this will break compatibility with xorgxrdp. 

See about 80 lines below. No worries but I need to take care about xrdp and xorgxrdp version in sync.
xorgxrdp v0.2.14 (to be released in August) must depend on v0.9.14.

Reply:
ref. https://github.com/neutrinolabs/xrdp/pull/542#discussion_r92231986

Reply:
Got it, it would be nice if that "comment" inside the source was a bit clearer on the why or have the file detail on how it is used or what the assumptions are.